odc/stats/plugins/_base.py

```diff
@@ -18,20 +18,24 @@ class StatsPluginInterface(ABC):
     def __init__(
         self,
         resampling: str = "bilinear",
-        input_bands: Sequence[str] = [],
-        chunks: Mapping[str, int] = {"y": -1, "x": -1},
+        input_bands: Sequence[str] = None,
+        chunks: Mapping[str, int] = None,
```

Comment:
These type hints are wrong.

Should be `Optional[Sequence[str]]` and `Optional[Mapping[str, int]]` respectively if they can take `None`.

Reply:
fixed in a new commit to the pr